webpack/ForemanInventoryUpload/Components/PageHeader/components/ToolbarButtons/ToolbarButtons.js

```diff
@@ -4,19 +4,21 @@ import SyncButton from '../SyncButton';
 import CloudConnectorButton from '../CloudConnectorButton';
 import './toolbarButtons.scss';
 import { selectSubscriptionConnectionEnabled } from '../../../InventorySettings/InventorySettingsSelectors';
+import { useIopConfig } from '../../../../../common/Hooks/ConfigHooks';
 
 const ToolbarButtons = () => {
   const subscriptionConnectionEnabled = useSelector(
     selectSubscriptionConnectionEnabled
   );
+  const useIopMode = useIopConfig();
```

Comment:
Variable names that start with `use` are assumed to be Hooks. For that reason I'd prefer to call it something else.

```suggestion
  const isIop = useIopConfig();
```